merlin/common/security/encrypt.py

```diff
@@ -52,11 +52,10 @@ def _get_key_path():
     except AttributeError:
         key_filepath = "~/.merlin/encrypt_data_key"
 
-    try:
-        key_filepath = os.path.abspath(os.path.expanduser(key_filepath))
-    except KeyError as e:
-        raise ValueError("Error! No password provided for RabbitMQ") from e
-    return key_filepath
+    if key_filepath is None:
+        raise ValueError("Error! No password provided for RabbitMQ")
+
+    return os.path.abspath(os.path.expanduser(key_filepath))
```

Comment:
Is an exception this may cause checked elsewhere?

Reply:
I don't think it's possible for `os.path.expanduser()` or `os.path.abspath()` to raise a KeyError so that's why I removed that try/except. The errors that are typically raised from these calls are `TypeError` and `OSError` in which case we probably want the exception raised anyways.

Reply:
OK